pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -66,20 +66,63 @@ type ClusterWorkspaceSpec struct {
 
 	// +optional
 	InheritFrom string `json:"inheritFrom,omitempty"`
+
+	// type defines properties of the workspace both on creation (e.g. initial
+	// resources and initially installed APIs) and during runtime (e.g. permissions).
+	//
+	// The type is a reference to a ClusterWorkspaceType in the same workspace
+	// with the same name, but lower-cased. The ClusterWorkspaceType existence is
```

Comment:
open question: do we want to inherit from the root? What about overriding? What about permission to create a ClusterWorkspaceType of a certain kind? Do we have to control an org admin in this? Or do we move that question to higher level accounting of some sort? cc @ncdc